src/Microsoft.OData.Core/UriParser/Aggregation/AggregateExpression.cs

```diff
@@ -8,43 +8,63 @@ namespace Microsoft.OData.UriParser.Aggregation
 {
     using Microsoft.OData.Edm;
     using Microsoft.OData.UriParser;
+    using System;
+    using System.Collections.Generic;
 
     /// <summary>
     /// A aggregate expression representing a aggregation transformation.
     /// </summary>
-    public sealed class AggregateExpression
+    public abstract class AggregateExpression

```

Comment:
Abstract classes usually have an abstract method for the derived class to implement.  Consider a protected default constructor if you don't want callers instantiating this class.